analyzer/src/test/kotlin/AnalyzerResultBuilderTest.kt

```diff
@@ -166,9 +166,13 @@ class AnalyzerResultBuilderTest : WordSpec() {
                     )
                 )
 
-                val serializedResult = result.toYaml().fromYaml<AnalyzerResult>().toYaml()
+                val serializedResult = result
+                    .toYaml()
+                    .fromYaml<AnalyzerResult>()
+                    .toYaml()
+                    .fromYaml<AnalyzerResult>()
 
-                serializedResult shouldBe serializedResult
+                serializedResult shouldBe result
```

Comment:
The original test compared YAML to YAML, which results in a textual diff on changes. Maybe this behavior could be maintained by

    val serializedResult = result.toYaml()
    serializedResult.fromYaml<AnalyzerResult>().toYaml() shouldBe serializedResult

This also avoids one additional `fromYaml` step.